src/psyclone/dynamo0p3.py

```diff
@@ -4759,6 +5091,11 @@ def cell_position(self):
             "Error: ArgOrdering.cell_position() must be implemented by "
             "subclass")
 
+    def cell_map(self):
+        ''' Add cell-map information (for inter-grid kernels) '''
```

Comment:
Shouldn't this have a doctring interface i.e. raises exception - sorry.

Reply:
Done, plus all the other 'abstract' methods for good measure.